ceh-2000/gpu_guru
# Implemented the unachievable ideal model

## Closes #2 

## Description
We want an upper-bound accuracy for our model architecture and given classification task. We achieve this by allowing a single model to train on all of the available training data to convergence. This is the `UnachievableIdeal` class.

## Testing:
1. Pull down the new code and open the README. Check the new instructions for running an "Unachievable Ideal" experiment.
2. Per the README, run the following command:
```
python3 main.py --algorithm central --dataset mnist --sample_ratio 0.1 --glob_epochs 5 --should_log True
``` 
3. Check that the results are logged by calling `tensorboard --logdir=runs` and navigating to [http://localhost:6006/](http://localhost:6006/).
